bitnami/zookeeper/values.yaml

```
@@ -272,6 +272,7 @@ tls:
     truststorePassword: ''
 
   quorum:
+    enabled: false
     ## Create self-signed TLS certificates. Currently only supports PEM certificates.
```

Could you add a small description for this parameter?
```suggestion
  quorum:
    ## Enable TLS for quorum protocol
    ##
    enabled: false

    ## Create self-signed TLS certificates. Currently only supports PEM certificates.
```